operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -837,7 +837,7 @@ def customizeNetworkAccessPoint(self, nap, listen_address):
     # FWIW there's theoretically no need to 'add' or 'replace' when empty
     #   since the runtime default is the server listen-address.
     nap_name=nap.getName()
-    if not (nap.getListenAddress() is None) and len(nap.getListenAddress()) > 0:
+    if not (nap.getListenAddress() is None) and len(nap.getListenAddress()) > 0 and not (nap_name.startswith('istio-')) and not (nap_name == 'T3Channel'):
```

Comment:
Why does the change suppress overriding the 'T3Channel'?  That seems arbitrary.

Reply:
The listen address has already been set to the right value: T3_PUBLIC_ADDRESS in create-domain.py and has been updated to 127.0.0.1 in istio-fix-domain.py. So I think it doesn't need to be overridden to service name in sitconfig, right?

Reply:
We need a reliable heuristic to detect if a nap needs overriding - 'T3Channel' is just an arbitrary name that happens to be in our main sample, where-as introspectDomain.py is intended to handle all NAP use cases.